src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -243,7 +243,7 @@ protected void addHighLevelBindings(final DocWorkUnit workUnit)
      */
     protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         final String tagFilterPrefix = getTagPrefix();
-        Arrays.stream(currentWorkUnit.getClassDoc().inlineTags())
+        Arrays.stream(currentWorkUnit.getClassDoc().tags())
```

Comment:
My aim here is to have tags similar to the javadoc `@see`, but customized. This will allow to remove from the help text this tags and even include them in the normal javadoc. For example:

```java
/**
 * This is the description of the documentation.
 * @barclay.type simple
 * @author magicDGS
 * @see OtherClass
 */
@DocumentedFeature
public class MyDocumentedClass {}
```

Maybe my change is not working (which is wierd, because it mimics a hack that I did in a custom project to include this feature), but my aim is to populate the "barclay.type" in the JSON, but ignoring it in the documentation text description. In Barclay, the test for round trip is testing the JSON, but the tags are not populated there anyway (I haven't change the expected files); we should either have a test JSON object or implement mocking tests for all this functionality.

I also notice another problem with the tags: if there are repeated tags, they are overridden. For instance, I have the situation when I want to add custom tags with "notes", "warnings", "caveats" and then populate them into the JSON to format them as a list. That's actually why I started with this: I do not want to have them embedded in the description (in the format `{@doc.warning be careful}`), but being able to process them with the `@doc.warning be careful` format to remove them from the doc test. Nevertheless, the problem is when two warnings should be separated, and then one overrides the other.

Maybe we should discuss about the tag-support and how it should look like in a different issue, and I should close this PR til we agree on the final shape of this functionality...